libandroid-navigation-ui/src/main/res/values/styles.xml

```diff
@@ -12,6 +11,9 @@
         <item name="routeScale">1.0</item>
         <item name="upcomingManeuverArrowColor">@color/mapbox_navigation_route_upcoming_maneuver_arrow_color</item>
         <item name="upcomingManeuverArrowBorderColor">@color/mapbox_navigation_route_upcoming_maneuver_arrow_border_color</item>
+
+        <!-- Expression styling -->
+        <item name="routeLineCap">@string/mapbox_navigation_route_line_cap_square</item>
```

Comment:
I think there is a typo here. Isn't `mapbox_navigation_route_line_cap_round` the default one?

Reply:
Yeah great catch @Guardiola31337